app/services/users.js

```diff
@@ -88,3 +89,20 @@ exports.createUserAdmin = userData => {
   userData.administrator = true;
   return upsert(userData);
 };
+
+exports.userAlbumsList = async req => {
+  try {
+    if (req.body.decode.administrator !== true) {
+      const user = await getUser(req.body.decode.email);
+      if (Number(user.id) !== Number(req.params.user_id)) {
```

Comment:
you can just compare `user.id !== req.params.user_id`. try it and tell me what happen please.

Reply:
if I do it that way I can't make the comparison, so I turn it into numbers so I can compare.